app/controllers/concerns/account_ab_testable.rb

```diff
@@ -1,6 +1,19 @@
+# frozen_string_literal: true
+
 module AccountAbTestable
+  extend ActiveSupport::Concern
+
   ACCOUNT_AB_CUSTOM_DIMENSION = 42
-  ACCOUNT_AB_TEST_NAME = "AccountExperiment".freeze
+  ACCOUNT_AB_TEST_NAME = "AccountExperiment"
+
+  ACCOUNT_SESSION_REQUEST_HEADER_NAME = "HTTP_GOVUK_ACCOUNT_SESSION"
+  ACCOUNT_SESSION_RESPONSE_HEADER_NAME = "GOVUK-Account-Session"
+  ACCOUNT_SESSION_DEV_COOKIE_NAME = "govuk_account_session"
```

Comment:
Why aren't we freezing these constants?

Reply:
I added `# frozen_string_literal: true` to the top of the file, which makes it automatic

Reply:
Ah, I had missed that!